library_samples/Python3/ocs_sample_library_preview/DataView/DataView.py

```diff
@@ -233,26 +310,43 @@ def fromDictionary(content):
             dataView.Description = content["Description"]
 
         if "Queries" in content:
-            dataView.Queries = DataViewQuery.fromDictionary(content["Queries"])
-
-        if "Mappings" in content:
-            dataView.Mappings = DataViewMappings.fromDictionary(
-                content["Mappings"])
-
-        if "IndexConfig" in content:
-            dataView.IndexConfig = DataViewIndexConfig.fromDictionary(
-                content["IndexConfig"]
-            )
-
-        if "IndexDataType" in content:
-            dataView.IndexDataType = content["IndexDataType"]
-
-        if "GroupRules" in content:
-            groupRules = content["GroupRules"]
-            if groupRules is not None and len(groupRules) > 0:
-                dataView.GroupRules = []
-                for value in groupRules:
-                    dataView.GroupRules.append(
-                        DataViewGroupRule.fromDictionary(value))
+            Queries = content["Queries"]
+            if Queries is not None and len(Queries) > 0:
+                dataView.Queries = []
+                for value in Queries:
+                    dataView.Queries.append(
+                        Query.fromDictionary(value))
+
+        if "FieldSets" in content:
+            FieldSets = content["FieldSets"]
+            if FieldSets is not None and len(FieldSets) > 0:
+                dataView.FieldSets = []
+                for value in FieldSets:
+                    dataView.FieldSets.append(
+                        FieldSet.fromDictionary(value))
+
+        if "Sectioners" in content:
+            Sectioners = content["Sectioners"]
+            if Sectioners is not None and len(Sectioners) > 0:
+                dataView.Sectioners = []
+                for value in Sectioners:
+                    dataView.Sectioners.append(
+                        Field.fromDictionary(value))
+
+        if "IndexTypeCode" in content:
+            # SdsTypeCode(content['IndexTypeCode'])
```

Comment:
Remove commented code?

Reply:
maybe?  I am hoping we fix it to be consistent...

Reply:
This is due to the difference in enums with Data Views then, I take it? I would at least maybe add a # TODO or something to mark that work needs to be done here?